modules/doc/content/help/faq/apptainer.md

```diff
@@ -0,0 +1,174 @@
+# Apptainer
```

Comment:
I think we should move this to something like:
```
Getting started->Installation->Apptainer
```
We already have an option for docker here:
https://mooseframework.inl.gov/getting_started/installation/index.html

We could have the same with apptainer.
What do you think?

Reply:
Maybe, but it is only useful for those with access to INL HPC machines. I wanted to group this in with that somehow.

Reply:
Would it be better to place this in `moose/doc/content/help/inl` then, in order to be directly beside the other HPC-related documents?

Reply:
I originally put it there, but that breaks Bison documentation.